tests/unit/config/config_test.py

```diff
@@ -573,6 +573,11 @@ def test_home_directory_with_driver_does_not_expand(self):
         }, working_dir='.')
         self.assertEqual(d['volumes'], ['~:/data'])
 
+    def test_volume_path_with_non_ascii_directory(self):
+        volume = '/Füü/data:/data'
+        container_path = config.resolve_volume_path(volume, ".", "test")
+        self.assertEqual(container_path, '/Füü/data:/data')
```

Comment:
I can confirm this test fails on python2 (windows and linux). 

I was able to fix it by chaining this to:

```python
volume = u'/Füü/data:/data'
```

and

```python
self.assertEqual(container_path, volume)
```

which matches a real run, because yaml.load() returns unicode strings (not bytes).